both/routes/userClass.js

```diff
@@ -0,0 +1,22 @@
+Router.map(function () {
+  this.route('userClass/create', {
+    path: '/userClass/create',
+    template: 'userClass'
+  //TODO: Permissions
+  });
+  this.route('userClass/:id/edit', {
```

Comment:
Added route to edit all classes.

Also, route /user/:id/edit already [exists](https://github.com/meteor-gazelle/meteor-gazelle/blob/master/both/routes/user.js#L28).

Resolved.